src/components/localization/language-switcher.js

```diff
@@ -13,11 +15,24 @@ const disabled_lang = ['ach']
 
 const LanguageSwitch = ({ i18n, is_high_nav, short_name }) => {
     const [language, setLanguage] = React.useState(i18n.language)
+    const client_information = useClientInformation()
 
     React.useEffect(() => {
         setLanguage(i18n.language)
     }, [i18n.language])
 
+    React.useEffect(() => {
+        if (!Cookies.get('lang_is_fixed')) {
+            if (client_information.preferred_language) {
+                const lang = client_information.preferred_language.toLowerCase()
+                if (lang !== language) {
+                    const replaced_lang = lang.replace('_', '-')
```

Comment:
Can you chain this on line 27 as well?

Reply:
I don't think I can chain this, language state has _ so i need to replace after the check